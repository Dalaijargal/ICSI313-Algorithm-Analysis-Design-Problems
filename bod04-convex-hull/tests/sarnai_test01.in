100
-14 60 77 -84 0 11 74 -100 -70 -77 85 -93 89 -56 -66 100 -50 34 -74 -97 -43 38 6 -35 -25 53 46 51 -7 90 56 99 87 -83 -55 16 -53 -78 37 48 -20 23 26 40 -17 -30 67 -69 30 33 19 42 93 52 -47 -72 66 13 -57 -49 -79 -41 43 -13 -87 70 -31 -80 -99 17 94 97 -40 -6 91 -4 -23 35 76 -67 10 24 -82 -88 84 79 96 98 61 31 -21 -45 14 58 88 83 86 62 5 64 -38 -15 -59 59 -44 -98 -36 80 -86 63 -33 7 -76 72 -91 -54 -28 69 -94 -39 -62 -3 25 -68 8 73 -24 -42 20 1 -37 -29 55 -63 -34 -89 -48 78 -60 50 -92 -85 41 -46 82 4 -65 21 -27 75 -71 3 44 -8 92 -64 2 -2 -90 36 -22 -19 -81 32 -16 -9 45 39 -75 -12 -18 15 -73 71 68 29 -5 22 9 65 -11 -95 54 18 -61 -26 47 81 27 -58 49 12 -1 -96 95 -51 -32 -52 -10 28