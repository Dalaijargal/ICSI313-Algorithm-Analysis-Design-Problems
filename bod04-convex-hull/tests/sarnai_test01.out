(74.0, -100.0), (85.0, -93.0), (-66.0, 100.0), (-44.0, -98.0), (-74.0, -97.0), (56.0, 99.0), (-87.0, 70.0), (-99.0, 17.0), (-92.0, -85.0), (96.0, 98.0), (92.0, -64.0), (95.0, -51.0),